platform/marvell-armhf/platform.conf

```diff
@@ -162,10 +246,17 @@ mount_partition() {
     }
 
     echo "Mounting $demo_dev on $demo_mnt "
-    trap_push "${onie_bin} umount /dev/ubi0_0|| true"
-    mount -t ubifs /dev/ubi0_0 $demo_mnt || {
-        echo "Failed"
-    }
+    if [ "$PLATFORM" = "armhf-marvell_et6448m_52x-r0" ]; then
+        trap_push "${onie_bin} umount /dev/ubi0_0|| true"
+        mount -t ubifs /dev/ubi0_0 $demo_mnt || {
+            echo "mount -t ubifs /dev/ubi0_0 $demo_mnt Failed"
```

Comment:
>Failed [](start = 55, length = 6)

If this is a failure, exit?

Reply:
on failure, trap push will take care as script is enabled with -e

Reply:
`mount ... || { echo "..." }` will always succeed. I see your message said "Failed", so I guess you need `exit` after `echo`.

---
In reply to: [526622606](https://github.com/Azure/sonic-buildimage/pull/5749#discussion_r526622606) [](ancestors = 526622606)

Reply:
added exit